app/components/provincie-select.js

```diff
@@ -42,8 +42,8 @@ export default Component.extend({
 
   actions: {
     changeSelected(selected) {
-      this.set('selected', selected);
-      this.onSelectionChange(selected && selected.id);
+      this.set('selected', selected)
```

Comment:
Any reason why the `;` has been removed?

Reply:
No